pyproject.toml

```diff
@@ -1,43 +1,57 @@
-[tool.black]
```

Comment:
I think you deleted the black/isort/ruff sections of this file. They need to come back.